daprdocs/content/en/concepts/dapr-services/scheduler.md

```diff
@@ -65,11 +65,11 @@ For non-client side errors, for example, when a job cannot be sent to an availab
 
 The Scheduler service Docker container is started automatically as part of `dapr init`. It can also be run manually as a process if you are running in [slim-init mode]({{< ref self-hosted-no-docker.md >}}).
 
-## Kubernetes mode
+The Scheduler can be run in both high availability (HA) and non-HA modes in self-hosted deployments. However, non-HA mode is not recommended for production use. If switching between non-HA and HA modes, the existing data directory must be removed, which will result in loss of jobs and actor reminders. Ensure you [run a back up]({{< ref "#back-up-and-restore-scheduler-data" >}}) before making this change to avoid losing data.
```

Comment:
```suggestion
The Scheduler can be run in both high availability (HA) and non-HA modes in self-hosted deployments. However, non-HA mode is not recommended for production use. If switching between non-HA and HA modes, the existing data directory must be removed, which results in loss of jobs and actor reminders. [Run a back-up]({{< ref "#back-up-and-restore-scheduler-data" >}}) before making this change to avoid losing data.
```